<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorAtemKeyerStateWidget</class>
 <widget class="QWidget" name="InspectorAtemKeyerStateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>156</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutMixerStep">
     <item>
      <spacer name="horizontalSpacerMixerStep">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>62</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelMixerStep">
       <property name="text">
        <string>M / E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxMixerStep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutKeyer">
     <item>
      <spacer name="horizontalSpacerKeyer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelKeyer">
       <property name="text">
        <string>Keyer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxKeyer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInput">
     <item>
      <spacer name="horizontalSpacerState1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelState">
       <property name="text">
        <string>On AIR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxState">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerState2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerInput">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>labelKeyer</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorAtemKeyerStateWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxState</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorAtemKeyerStateWidget</receiver>
   <slot>stateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxKeyer</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorAtemKeyerStateWidget</receiver>
   <slot>keyerChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxMixerStep</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>InspectorAtemKeyerStateWidget</receiver>
   <slot>mixerStepChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stateChanged(int)</slot>
  <slot>keyerChanged(int)</slot>
  <slot>triggerOnNextChanged(int)</slot>
  <slot>mixerStepChanged(int)</slot>
 </slots>
</ui>
